### PR TITLE
Update slack.svg

### PR DESCRIPTION
**Pull request by ashygee**
Updating the slack icon to new logo

### CODE REVIEW

**Comment by ashygee** [comment]
Updated slack logo per conversation from #516

File: icons/slack.svg [hunk 1]
```diff
@@ -4,14 +4,7 @@
...
... 14.1193 17 15.5 17L15.5 15ZM21 14.5C21 14.7761 20.7761 15 20.5 15L20.5 17C21.8807 17 23 15.8807 23 14.5L21 14.5ZM15.5 12C14.1193 12 13 13.1193 13 14.5L15 14.5C15 14.2239 15.2239 14 15.5 14L15.5 12ZM20.5 14C20.7761 14 21 14.2239 21 14.5L23 14.5C23 13.1193 21.8807 12 20.5 12L20.5 14ZM15 20.5V19H13V20.5H15ZM14 20H15.5V18H14V20ZM15.5 21C15.2239 21 15 20.7761 15 20.5L13 20.5C13 21.8807 14.1193 23 15.5 23L15.5 21ZM16 20.5C16 20.7761 15.7761 21 15.5 21L15.5 23C16.8807 23 18 21.8807 18 20.5L16 20.5ZM18 20.5C18 19.1193 16.8807 18 15.5 18L15.5 20C15.7761 20 16 20.2239 16 20.5L18 20.5ZM8.5 10H3.5V12H8.5V10ZM8.5 7L3.5 7L3.5 9L8.5 9L8.5 7ZM3.5 7C2.11929 7 1 8.11929 1 9.5L3 9.5C3 9.22386 3.22386 9 3.5 9L3.5 7ZM11 9.5C11 8.11929 9.88071 7 8.5 7L8.5 9C8.77614 9 9 9.22386 9 9.5L11 9.5ZM8.5 12C9.88071 12 11 10.8807 11 9.5L9 9.5C9 9.77614 8.77614 10 8.5 10L8.5 12ZM3.5 10C3.22386 10 3 9.77614 3 9.5L1 9.5C1 10.8807 2.11929 12 3.5 12L3.5 10ZM9 3.5V5H11V3.5H9ZM10 4H8.5V6H10V4ZM8.5 3C8.77614 3 9 3.22386 9 3.5L11 3.5C11 2.11929 9.88071 1 8.5 1L8.5 3ZM8 3.5C8 3.22386 8.22386 3 8.5 3L8.5 1C7.11929 1 6 2.11929 6 3.5L8 3.5ZM6 3.5C6 4.88071 7.11929 6 8.5 6L8.5 4C8.22386 4 8 3.77614 8 3.5L6 3.5Z" fill="black"/>
```
Comment (colebemis) [inline icons/slack.svg]: Looks like you may have converted stroke to outlines here.

Comment (colebemis) [reply icons/slack.svg]: Can you export a stroke version instead?

Comment (colebemis) [reply icons/slack.svg]: Also, don't forget to run `npm run build` after adding a new svg 😄

Comment (ashygee) [reply icons/slack.svg]: weird, i didn't flatten it but i'll go ahead and double check later.

**Comment by colebemis** [comment]
Looks like there's a 2px gap in between the top circle and line (see pink arrow). I think that top circle just needs to be moved 1px to the right
image

**Pull request merged**